frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -28,7 +28,7 @@ import {
 } from "metabase/selectors/user";
 
 import * as actions from "../actions";
-import View from "../components/view/View";
+import { View } from "../components/view/View";
```

Comment:
Maybe add `index.ts` file to `View`?

Reply:
ah yeah that makes sense

Reply:
I'll do that in a separate PR to let the tests run actually